taipy/gui/_renderers/factory.py

```diff
@@ -618,6 +619,19 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "alert": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="AlertComponent",  
+            attributes=attrs,
+        )
+        .set_value_and_default(with_update=True)  # Enable dynamic updates for the 'message'
```

Comment:
👍I think true is the default value ?

You should give it a type `var_type= PropertyType.dynamic_string`